draft-barnes-acme.md

```diff
@@ -307,6 +308,31 @@ Authorization and challenge objects can also contain error information to indica
 
 TODO: Flesh out errors and syntax for them
 
+## Directory
+
+In order to help clients configure themselves with the right URLs for each ACME
+operation, ACME servers provide a directory object. This should be the root URL
+with which clients are configured. It has this structure:
+
+newRegistration (required, string): URL to the new-registration resource.
```

Comment:
I realize we use this for other ACME objects, but this notation is a bad fit for this purpose.  Better to just
* Say that the directory is a JSON dictionary, whose keys are from a standard list of functions, and whose values are URIs for those functions (as strings)
* Define the list of functions, with cross references to the relevant sections